### PR TITLE
HEEDLS-480 Fix bug throwing exception when nothing is input in textbox

### PR DESCRIPTION
**Pull request by ibrahimmunir14**

### CODE REVIEW

**Review by AlexJacksonDS — APPROVED** [review]
Looks good, I assume the line endings are correct? The whole file is showing as changed

**Comment by ibrahimmunir14** [comment]
> Looks good, I assume the line endings are correct? The whole file is showing as changed

All CRLF

File: DigitalLearningSolutions.Web/Attributes/NoWhitespaceAttribute.cs [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
-        {
-            if (value.ToString().Any(char.IsWhiteSpace))
-            {
-                return new ValidationResult(errorMessage); 
-            }
-            return ValidationResult.Success;
-        }
-    }
-}
+﻿namespace DigitalLearningSolutions.Web.Attributes
```
Comment (stellake) [inline DigitalLearningSolutions.Web/Attributes/NoWhitespaceAttribute.cs:1]: Is it worth writing some unit tests for this?

**Review by stellake — APPROVED** [review]
Looks great! 👍

**Pull request merged**